int32 landmark_count

float64[100] x
float64[100] y
int32[100] range
int32[100] bearing
int32[100] id
